src/install_linux.ts

```diff
@@ -35,7 +35,12 @@ export function install(config: Config): Promise<Installed> {
         if (config.version === 'stable') {
             return downloadStableNeovim('linux', config.token);
         } else {
-            return downloadNeovim(config.version, 'linux');
+            try {
+                return downloadNeovim(config.version, 'linux');
+            } catch (e) {
+                core.debug(`Neovim download failure: ${e.message}.`);
```

Comment:
```suggestion
                const err = const err = ensureError(e);
                core.debug(`Neovim download failure: ${err.message}.`);
```

`ensureError()` is necessary to make TypeScript compiler happy.